### DISCUSSION TITLE
/articles/sharepoint-framework-domain-isolated-webpart-retirement/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#241] [Comments]
Microsoft is at it again… building off all the retirement notices in 2023, Microsoft is as it again killing off a SharePoint Framework (SPFx) feature.
https://www.andrewconnell.com/articles/sharepoint-framework-domain-isolated-webpart-retirement/

**Comment by PeterWCox** [comment]
Its been a while doing SPFx work, but I always found Isolated Webparts to be an awful developer experience. Often a lot of the CSS broke, or there were multiple 'gotchas' requiring lots of messing around on google and stackoverflow.
